full/src/main/java/apoc/metrics/Metrics.java

```diff
@@ -43,7 +43,7 @@ public StoragePair(String setting, File dir) {
         public static StoragePair fromDirectorySetting(String dir) {
             if (dir == null) return null;
 
-            String configLocation = apocConfig().getString("apoc." + dir, null);
+            String configLocation = apocConfig().getString(dir, null);
```

Comment:
Scusa ma per retrocompatibilità non sarebbe meglio così?
```suggestion
            String configLocation = apocConfig().getString(dir, apocConfig().getString("apoc." + dir, null));
```

Reply:
In realtà credo sia proprio sbagliato rimanere “apoc.”, non penso sia mai funzionato in precedenza così com’è,
perché questo metodo viene usato solo mentre ciclo `FileUtils.NEO4J_DIRECTORY_CONFIGURATION_SETTING_NAMES`
che contiene tutte le configurazioni di neo4j che cominciano con `dbms.directories.*`.
Quindi non penso abbia senso cercare una configurazione tipo `apoc.dbms.directories.import`.

Reply:
Ok quando fai la PR per favore fatti una auto review e aggiungi una nota per David (chiedi a lui di farti la review)